android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/app/ApplicationConfiguration.kt

```diff
@@ -31,7 +31,8 @@ data class ApplicationConfiguration(
   var applicationName: String = "",
   var appLogoIconResourceFile: String = "ic_default_logo",
   var patientTypeFilterTagViaMetaCodingSystem: String = "",
-  var count: String = ConfigurationRegistry.DEFAULT_COUNT
+  var count: String = ConfigurationRegistry.DEFAULT_COUNT,
+  var taskFilterTagViaMetaCodingSystem: String = ""
```

Comment:
Make the default the current tag. You can add it as a constant in ConfigurationRegistry.